packages/create-plugin/src/commands/provisioning.command.ts

```diff
@@ -0,0 +1,22 @@
+import glob from 'glob';
+import { TEMPLATE_PATHS, TEXT } from '../constants';
+import { compileProvisioningTemplateFile, getTemplateData } from '../utils/utils.templates';
+import { confirmPrompt, printMessage, printSuccessMessage } from '../utils/utils.console';
+
+export const provisioning = async () => {
+  try {
+    if (await confirmPrompt(TEXT.updateProvisioning)) {
+      const provisioningSpecificFiles = glob.sync(`${TEMPLATE_PATHS.provisioning}/**`, { dot: true });
+
+      provisioningSpecificFiles.map((file) => {
+        compileProvisioningTemplateFile(file, getTemplateData());
+      });
```

Comment:
since we are not using the return of map, I think we can use a for each here:
```ts
      provisioningSpecificFiles.forEach((file) => {
        compileProvisioningTemplateFile(file, getTemplateData());
      });
```